moj-node/server/repositories/hubFeaturedContent.js

```diff
@@ -1,59 +1,15 @@
-const R = require('ramda');
 
-const { sanitizeTruncateText } = require('../utils/index');
+const { parseResponse } = require('../utils/index');
```

Comment:
Instead of `parseResponse` perhaps we should  call it `parseHubContentResponse`

Reply:
Yeah agreed. Says what it's doing, I"ll make the change